server/insights/src/main/java/org/infinispan/server/insights/report/InfinispanTopReport.java

```diff
@@ -0,0 +1,31 @@
+package org.infinispan.server.insights.report;
+
+import java.util.Map;
+
+import org.infinispan.manager.CacheManagerInfo;
+
+import com.redhat.insights.InsightsException;
+import com.redhat.insights.InsightsSubreport;
+import com.redhat.insights.config.InsightsConfiguration;
+import com.redhat.insights.core.TopLevelReportBaseImpl;
+import com.redhat.insights.logging.InsightsLogger;
+
+public class InfinispanTopReport extends TopLevelReportBaseImpl {
+
+   private final CacheManagerInfo cacheManagerInfo;
+
+   public InfinispanTopReport(InsightsLogger logger, InsightsConfiguration config, CacheManagerInfo cacheManagerInfo,
+                              Map<String, InsightsSubreport> subReports) {
+      super(logger, config, subReports);
+      this.cacheManagerInfo = cacheManagerInfo;
+   }
+
+   @Override
+   protected String getIdentificationName() {
+      try {
+         return super.getIdentificationName();
+      } catch (InsightsException ex) {
+         return "Infinispan Server " + cacheManagerInfo.getNodeName();
```

Comment:
Do we want to completely ignore the exception?

Reply:
According to the EAP Insights integration code it seems to be safe, see https://github.com/jbossas/jboss-eap7/blob/7.4.x/insights/src/main/java/org/jboss/eap/insights/report/JBossTopLevelReport.java.
Since I haven't received any feedback from the Red Hat Insights guys I don't know if it is OK.
I just copied and adapted what has been done by the EAP team.

Reply:
Also use `Version.brandName()`